pkg/controller/admissionchecks/multikueue/workload.go

```diff
@@ -52,6 +52,10 @@ var (
 	errNoActiveClusters = errors.New("no active clusters")
 )
 
+const (
+	MultiKueueGCLabelKey = "kueue.x-k8s.io/multikueue-gc"
```

Comment:
I think there is no reason to couple the label name to GC, is there? It might be useful in other contexts, say when debugging to answer questions like: "what is the number of workloads managed by multikueue", "are there any workloads not managed by multikueue". Such questions aren't directly related to GC.

Reply:
I think we should try to not delete workloads that ware not created by us ("us" being the instance of kueue running the garbage collection).

Reply:
Sure, I'm just suggesting the name of the label should be `kueue.x-k8s.io/multikueue` (not coupled to GC)

Reply:
changed to ` kueue.x-k8s.io/multikueue-origin`